applications/vessel-history/public/locales/source/translations.json

```diff
@@ -236,9 +236,14 @@
       "0": "Not listed",
       "1": "previously listed",
       "2": "currently listed"
+    },
+    "authorizations": {
```

Comment:
Apparently "authorizations" translation was missing but it was in use. That's why the title of the popup is wrong:
![image](https://user-images.githubusercontent.com/2205831/135524995-3fa21e0b-811d-443f-a310-26a4ef411d9b.png)

Use other key instead, eg: `auths`
```suggestion
    "auths": {
```

Reply:
wow I didn't see that